### ISSUE TITLE
Grids are not in same position with their headers

### ISSUE THREAD

**Issue by vahidhashemi** [fullcalendar/fullcalendar#6154]
### Reduced Test Case
not reproducible with a link

### Bug Description

This bug is more like a Heisenbug, I mean I can see it but I don't know how to reproduce it, hence I share couple of screenshots. Hope you you can fix the problem.

Basically the grids in the main view get displaced a little bit, as if they are not on the same columns with their headers.

What it does? when a user try to select a time, he will select a wrong time unintentionally. In our case the time-span is 24 hours so this little space gives us a time with +-2 hours error.

Browser : Chrome 88
Resolution : 1920x1080
OS : Windows10

Haven't seen this bug in Firefox
By the way I couldn't reproduce this bug on my MBP 15inch.

### Screenshots

Fullscreen view (it is not much clear)
![bug-1](https://user-images.githubusercontent.com/1707050/107323373-658b7900-6abb-11eb-805d-063a2afd9fab.jpg)

Bug Zoom in
![bug2](https://user-images.githubusercontent.com/1707050/107323401-720fd180-6abb-11eb-9b5c-061e4c838862.jpg)

Fixed Zoom in
![bug-fix3](https://user-images.githubusercontent.com/1707050/107323422-7c31d000-6abb-11eb-85c8-de5b5d3728d1.jpg)

**Comment by HennebergITM** [comment]
I had this problem as well, but I found a small fix that worked for me. When the page is loaded, I simple call the scrollToTime-function on the API. I then wrapped it in a setTimeout, and that has fixed it for me forever.
(This particular example is from Angular, where I put it inside ngAfterViewInit)

![image](https://user-images.githubusercontent.com/78542190/107337833-25c49180-6abb-11eb-8b66-d3ca1c750b0e.png)
